Batch File/Batch File.sublime-syntax

```diff
@@ -34,6 +35,12 @@ contexts:
     - match: '<&?|>[&>]?'
       scope: keyword.operator.redirection.dosbatch
 
+  labels:
+    - match: '(?i):(\w+)\r?\n'
+      captures:
+        0: meta.function.dosbatch
+        1: entity.name.function.dosbatch
```

Comment:
Spaces and `+=,;:` are not allowed to be the first character in a label after the `:` but they can be using thereafter. Labels can also contain special characters (even non-ascii).

Should we differentiate between label definitions & label calls?

```batchfile
goto :label
call :label

:label
```